src/libraries/Microsoft.NETCore.Platforms/pkg/Microsoft.NETCore.Platforms.pkgproj

```diff
@@ -5,14 +5,25 @@
     <!-- We don't need to harvest the stable packages to build this -->
     <HarvestStablePackage>false</HarvestStablePackage>
     <PackageDescription>Provides runtime information required to resolve target framework, platform, and runtime specific implementations of .NETCore packages.</PackageDescription>
+    <InferRuntimeIdentifiers Condition="'$(DotNetBuildFromSource)' == 'true'">$(InferRuntimeIdentifiers);$(OutputRID)</InferRuntimeIdentifiers>
```

Comment:
It looks like we have both PackageRID and OutputRID since the runtime consolidation.  I'm not sure why we have two: they both seem to try to be accomplishing the same thing.  I use OutputRID here since that's what installer appears to use.

Reply:
I believe @jkoritzinsky is who has more context about `OutputRID` vs `PackageRID` but it seems like installer uses `PackageRID` to restore tooling and stuff and `OutputRID` is the rid to produce the package for.

Reply:
> installer uses PackageRID to restore tooling and stuff

I don't think its tooling.  I think installer uses this to for selecting packages from CoreCLR / Libraries (was more relevant when those were separate repos).  I'm 90% sure that its wrong to have two different properties here.

Reply:
Yep something like that was I couldn't remember exactly. I think we should just use PackageRID and remove OutputRID